packages/lib/formulas/logarithm/handler.ts

```diff
@@ -0,0 +1,16 @@
+import type { FormulaHandler } from '@toddledev/core/dist/types'
+
+const handler: FormulaHandler<number> = ([a]) => {
+  // Check if input is not a number or is null
+  if (typeof a !== 'number' || a === null || Number.isNaN(a)) {
```

Comment:
Technically, the second condition is obsolete, since `a` can't be a `null` if its type is `number` ;)
```suggestion
  if (typeof a !== 'number' || Number.isNaN(a)) {
```